web-app/components/pages/login/login.jsx

```diff
@@ -133,6 +182,16 @@ export const Login = () => {
                             <LoginButton variant="contained" size="medium" color="primary" type="submit" disableElevation>
                                 Login
                             </LoginButton>
+                            <Backdrop sx={(theme) => ({ color: '#fff', zIndex: theme.zIndex.drawer + 1 })} open={open}>
```

Comment:
Normally, we use 6 characters for hex color code. Using `#fff` probably works, but please do it better with `#FFFFFF` (the 6 characters and uppercase)